pylearn2/sandbox/nlp/models/tests/test_mlp.py

```diff
@@ -0,0 +1,15 @@
+'''
+Created on Jan 17, 2015
+
+@author: Minh Ngoc Le
```

Comment:
You should probably follow the format used everywhere else (e.g. [here](https://github.com/lisa-lab/pylearn2/blob/master/pylearn2/models/model.py#L1-L7)).